docs/javascripts/imagelightbox.js

```diff
@@ -160,8 +160,56 @@
                     $arrows.css('display', 'block');
                 }
             },
+            _pushHistory = function () {
+                var query = document.location.search;
+                var newParam = 'imageLightboxIndex=' + targetIndex;
```

Comment:
put newParam and newQuery into the "if (query)" block

Reply:
Can't do that - this value of `newQuery` is used when there is no query at this time (the `if` doesn't get triggered)